### ISSUE TITLE
Why does handsontable-pro/react not have a handsontable-pro.d.ts?

### ISSUE THREAD

**Issue by babyWatchmen** [handsontable/react-handsontable#124]

**Comment by AMBudnik** [comment]
Hey @babyWatchmen 

the wrappers are using the following definition file https://github.com/handsontable/handsontable/blob/master/handsontable.d.ts

we have one file for all the wrappers

**Comment by babyWatchmen** [comment]
Thank you, but there's a phenomenon.


![371545652459_ pic_hd](https://user-images.githubusercontent.com/24561833/50398914-54396280-07b6-11e9-9c48-27d7c04f5dc1.jpg)
![381545652459_ pic_hd](https://user-images.githubusercontent.com/24561833/50398915-569bbc80-07b6-11e9-82bb-16f07450ea70.jpg)


The @handsontable-pro index.d.ts file from my clone on GitHub uses handsontable-pro, which is incorrect. When I changed to handsontable, I used it correctly.
